### PR TITLE
chore: remove reference to googleapis

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: asset/pom.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
   <artifactId>cloudasset-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google Cloud Asset Inventory Snippets</name>
-  <url>https://github.com/googleapis/java-asset</url>
```
Comment (Shabirmean) [inline asset/pom.xml:8]: Should we point to the directory in this repo? 
https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/asset

File: batch/snippets/pom.xml [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google Google Cloud Batch Snippets</name>
-  <url>https://github.com/googleapis/java-batch</url>
```
Comment (Shabirmean) [inline batch/snippets/pom.xml:10]: Should we link this to: https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/batch

File: recaptcha_enterprise/pom.xml [hunk 3]
```diff
@@ -5,7 +5,6 @@
   <artifactId>recaptcha-enterprise-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google reCAPTCHA Enterprise Snippets</name>
-  <url>https://github.com/googleapis/java-recaptchaenterprise</url>
```
Comment (Shabirmean) [inline recaptcha_enterprise/pom.xml:8]: Should we link it to:
https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/recaptcha_enterprise

File: talent/snippets/pom.xml [hunk 4]
```diff
@@ -5,7 +5,6 @@
   <artifactId>talent-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google Talent Solution Snippets</name>
-  <url>https://github.com/googleapis/java-talent</url>
```
Comment (Shabirmean) [inline talent/snippets/pom.xml:8]: Relink to this repo?

File: tasks/snippets/pom.xml [hunk 5]
```diff
@@ -5,7 +5,6 @@
   <artifactId>cloudtasks-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google Cloud Tasks Snippets</name>
-  <url>https://github.com/googleapis/java-tasks</url>
```
Comment (Shabirmean) [inline tasks/snippets/pom.xml:8]: Relink to this repo?

**Review by Shabirmean — APPROVED** [review]
LGTM! Can we add the URL t point to the samples folders in this repo?

**Comment by averikitsch** [comment]
> LGTM! Can we add the URL t point to the samples folders in this repo?

I'm against this idea due to the potential future updates and the limited benefits

**Pull request merged**